docs/community-operator-pipeline.md

```diff
@@ -0,0 +1,40 @@
+# community-operator-pipeline
+
+The community operator pipeline provides a platform for openshift community contributors to validate, share, and distribute openshift operators.
+
+**community-hosted-pipeline**
+
+The community hosted pipeline is used to certify the Operator bundles.
+
+It’s an additional layer of validation that has to run within the Red Hat infrastructure. It is intended to be triggered upon the creation of a bundle pull request and successfully completes with merging it. As part of the community hosted pipeline, the operator submitted in the PR will be tested via preflight trigger. 
+
+The community hosted pipeline run results will be posted in the github PR comment.
+
+![community-hosted-pipeline](images/op_test_pr.png)
+
+**community-release-pipeline**
+
+The community release pipeline is responsible for releasing a bundle image which has passed certification.
```

Comment:
I think we should try to avoid the term "certification" here. It could cause confusion with the "proper" ISV operator certification workflow.